crates/rspack_binding_options/src/options/raw_module/js_value_ref.rs

```diff
@@ -0,0 +1,74 @@
+use std::{
```

Comment:
This file would be quite general. Would you please move this to `rspack_napi_shared/src/js_values` ?